src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -103,19 +106,31 @@ class CompilerContainer {
    */
   compilationButton (name) {
     if (!name) name = ''
-    var displayed = name === '' ? '<no file selected>' : name
-    var el = yo`
-    <div class="px-2 mt-2 pb-0 d-flex">
-      <button class="btn btn-primary btn-block ${name === '' ? 'disabled' : ''}" title="Compile" onclick="${this.compile.bind(this)}">
+    const displayed = name === '' ? '<no file selected>' : name
+    const disabled = name === '' ? 'disabled' : ''
+    const compileBtn = yo`
+      <button id="compileBtn" class="btn btn-primary btn-block ${disabled}" title="Compile" onclick="${this.compile.bind(this)}">
         <span>${this._view.compileIcon} Compile ${displayed}</span>
       </button>
-    </div>`
-    if (name === '') {
-      el.setAttribute('disabled', 'true')
-    }
+    `
+    let el = yo`
+      <div class="px-2 mt-2 pb-0 d-flex">
+        ${compileBtn}
+      </div>
+    `
     return el
   }
 
+  _disableCompileBtn (disable) {
```

Comment:
Can you add a comment on the method ? Also as the param is a boolean, I would recommend `isDisabled` or `shouldDisable`
```javascript
/**
 * Toggle the compile button
 * @param {boolean} isDisabled Should disable or enable
 */
```

Reply:
I don't think it needs comment, the name is clear enough.